packages/smooth_app/lib/pages/user_preferences_page.dart

```diff
@@ -196,3 +167,33 @@ class UserPreferencesPage extends StatelessWidget {
     return result;
   }
 }
+
+class _AttributeListTile extends StatelessWidget {
+  const _AttributeListTile(this.attribute);
+
+  final Attribute attribute;
+
+  @override
+  Widget build(BuildContext context) {
+    final ProductPreferences preferences = context.watch<ProductPreferences>();
+    final bool isImportant =
+        preferences.getImportanceIdForAttributeId(attribute.id!) !=
+            PreferenceImportance.ID_NOT_IMPORTANT;
+    return SwitchListTile(
+      title: Text(attribute.name!),
+      secondary: SvgCache(attribute.iconUrl, width: 40),
+      value: isImportant,
+      onChanged: (bool value) => _setImportance(context, value),
+    );
+  }
+
+  void _setImportance(BuildContext context, bool isImportant) {
+    final ProductPreferences preferences = context.read<ProductPreferences>();
+    preferences.setImportance(
+      attribute.id!,
+      isImportant
+          ? PreferenceImportance.ID_IMPORTANT
```

Comment:
"important" is not "very important" and not "mandatory".
I'm not a big fan of a binary choice - instead of the previous 4 distinct values - though I understand that UI-wise a binary choice is more standard.
Perhaps "mandatory" would make more sense than "important"?
Other option (possibly in another PR): a 4-value button.